setup.py

```diff
@@ -15,7 +15,6 @@
 'pyelasticsearch',
 'celery',
 'pyyaml',
-'django-celery',
```

Comment:
Oh yes, good point! (That's because at first I took out django-celery completely, and then realised it would be helpful to still have for the results backend...)